nautobot/extras/views.py

```diff
@@ -970,9 +973,14 @@ def get(self, request, class_path=None, slug=None):
 
     def post(self, request, class_path=None, slug=None):
         job_model = self._get_job_model_or_404(class_path, slug)
+        template_name = "extras/job.html"
+
+        job_cls = job_model.job_class()
```

Comment:
Below we have: `... if job_model.job_class is not None ...` which is ensuring this method is defined.

We should have proper handling for this even though it is hoisted up here.

Reply:
this was similar to how it was done originally, but i've added exception handling to it inline with the `get` method